x-pack/plugins/enterprise_search/common/types/engines.ts

```diff
@@ -14,15 +14,9 @@ export interface EnterpriseSearchEnginesResponse {
 }
 
 export interface EnterpriseSearchEngine {
```

Comment:
are we hydrating `indices` with the fully index data for the `list` request?

Reply:
We are fetching indices data (`stats`) from ES directly. Logic added in this file. 
https://github.com/elastic/kibana/blob/7eb14bf51a9912f4bbf82afe8018d2ee3b3a9120/x-pack/plugins/enterprise_search/server/lib/engines/fetch_indices_stats.ts